Source/Libraries/NetCore/NetCoreMessages.cs

```diff
@@ -27,10 +27,8 @@ public NetCoreSimpleMessage(string _Type)
     [MemberConfig(TargetMember.All)]
     public class NetCoreAdvancedMessage : NetCoreMessage
     {
-        public string ReturnedFrom;
-        public bool Priority = false;
-        public Guid? requestGuid = null;
-        public object objectValue = null;
+        public Guid? requestGuid { get; set; } = null;
+        public object objectValue { get; set; } = null;
 
```

Comment:
Should I revert this code for the same serialization reason?

Reply:
Yeah. Missed this one.

Reply:
Done